armv7hnl-desktop-omv-defconfig

```diff
@@ -11415,7 +11415,7 @@ CONFIG_PREEMPTIRQ_DELAY_TEST=m
 # CONFIG_SAMPLES is not set
 CONFIG_ARCH_HAS_DEVMEM_IS_ALLOWED=y
 CONFIG_STRICT_DEVMEM=y
-# CONFIG_IO_STRICT_DEVMEM is not set
+CONFIG_IO_STRICT_DEVMEM=y
```

Comment:
Not going to work after https://github.com/OpenMandrivaAssociation/kernel-release/pull/149, since you disable DEVMEM.